app/request_list.tsx

```diff
@@ -81,8 +85,10 @@ const RequestList = (props: RequestObj) => {
       showPaginationTop={false}
       showPaginationBottom={true}
       pageSize={10}
+      freezeWhenExpanded={true}
       SubComponent={row => (
-        <InterceptForm rowProps={row} handleIntercept={props.handleIntercept} />
+        <InterceptForm freezeWhenExpanded={true} rowProps={row} handleIntercept={props.handleIntercept} handleStatusCodeChange={props.handleStatusCodeChange} handleRespTextChange={props.handleRespTextChange} ResponseText={props.ResponseText}
```

Comment:
use defaultProp and instead of `freezeWhenExpanded={true}` just use `freezeWhenExpanded`